code/modules/research/xenobiology/crossbreeding/burning.dm

```diff
@@ -0,0 +1,426 @@
+	for(var/mob/living/simple_animal/slime/S in view(7, get_turf(user)))
+		if(user in S.Friends)
+			var/friendliness = S.Friends[user]
+			S.Friends = list()
+			S.Friends[user] = friendliness
+		else
+			S.Friends = list()
+		S.rabid = 1
+		S.visible_message("<span class='danger'>The [S] is driven into a dangerous frenzy!</span>")
+	..()
+
+/obj/item/slimecross/burning/green
+	colour = "green"
+
+/obj/item/slimecross/burning/green/do_effect(mob/user)
+	var/which_hand = "l_hand"
+	if(!(user.active_hand_index % 2))
+		which_hand = "r_hand"
+	var/mob/living/L = user
+	if(!istype(user))
+		return
+	var/obj/item/held = L.get_active_held_item() //This should be itself, but just in case...
+	L.dropItemToGround(held)
+	var/obj/item/nullrod/armblade/slime/blade = new(user)
+	if(!L.put_in_hands(blade))
+		qdel(blade)
+		user.visible_message("<span class='warning'>[src] melts onto [user]'s arm, boiling the flesh horribly!</span>")
+	else
+		user.visible_message("<span class='danger'>[src] sublimates the flesh around [user]'s arm, transforming the bone into a gruesome blade!</span>")
+	user.emote("scream")
+	L.apply_damage(30,BURN,which_hand)
+	..()
+
+/obj/item/slimecross/burning/pink
+	colour = "pink"
+
+/obj/item/slimecross/burning/pink/do_effect(mob/user)
+	user.visible_message("<span class='notice'>[src] shrinks into a small, gel-filled pellet!</span>")
+	new /obj/item/slimecrossbeaker/pax(get_turf(user))
+	..()
+
+/obj/item/slimecross/burning/gold
+	colour = "gold"
+
+/obj/item/slimecross/burning/gold/do_effect(mob/user)
+	user.visible_message("<span class='danger'>[src] shudders violently, and summons an army for [user]!</span>")
+	for(var/i in 1 to 3) //Less than gold normally does, since it's safer and faster.
+		var/mob/living/simple_animal/S = create_random_mob(get_turf(user), HOSTILE_SPAWN)
+		S.faction |= "[REF(user)]"
+		if(prob(50))
+			for(var/j = 1, j <= rand(1, 3), j++)
+				step(S, pick(NORTH,SOUTH,EAST,WEST))
+	..()
+
+/obj/item/slimecross/burning/oil
+	colour = "oil"
+
+/obj/item/slimecross/burning/oil/do_effect(mob/user)
+	user.visible_message("<span class='danger'>[src] begins to shake with rapidly increasing force!</span>")
+	addtimer(CALLBACK(src, .proc/boom), 50)
+
+/obj/item/slimecross/burning/oil/proc/boom()
+	explosion(get_turf(src), 2, 4, 4) //Same area as normal oils, but increased high-impact values by one each, then decreased light by 2.
+	qdel(src)
+
+/obj/item/slimecross/burning/black
+	colour = "black"
+
+/obj/item/slimecross/burning/black/do_effect(mob/user)
+	var/mob/living/L = user
+	if(!istype(L))
+		return
+	user.visible_message("<span class='danger'>[src] absorbs [user], transforming them into a slime!</span>")
+	var/mob/living/simple_animal/slime/transformedslime/S = new(get_turf(L))
+	S.originalbody = L
+	L.forceMove(S)
+	S.name = L.real_name
+	L.mind.transfer_to(S)
+	S.status_flags |= GODMODE
```

Comment:
Uh what. I'm going to assume you wanted to make the original body indestructible here. Better idea to me is to give them shapeshift spell that removes itself on untrasform so all the code is not duplicated here.

Reply:
I intentionally didn't use the shapeshift spell subtype, because I'd have had to override a lot more things than the code I used here. If you want me to use it, though, I can switch